src/sqlfluff/core/default_config.cfg

```diff
@@ -71,6 +71,7 @@ capitalisation_policy = consistent
 # Comma separated list of words to ignore for this rule
 ignore_words = None
 ignore_words_regex = None
+ignore_schemas = None
```

Comment:
Is this valid for L010? Or do you want to ignore all keywords capitalisation rules for all statements with a `FROM` clause using this schema? That's a MUCH bigger ask than I was understanding...

I thought you only wanted it for L014?

Reply:
I originally only wanted this for L014, but looking at the code, it seemed easier (and more consistent) to do it in the parent rule. It should work for all that descend from L010, as far as I can see.

Reply:
I don't think the wording `ignore_schema` fits for L010 (or any of its other descendent rules). That is only relevant for L014.

So either we should make this L014-specific or choose a better name (`enhance_ignores`?)

Personally I'd prefer this to be L014-specific.